tools/license-builder.sh

```diff
@@ -58,6 +59,7 @@ else
 fi
 
 addlicense "libuv" "deps/uv" "$(cat ${rootdir}/deps/uv/LICENSE)"
+addlicense "nghttp2" "deps/nghttp2" "$(cat ${rootdir}/deps/nghttp2/COPYING)"
```

Comment:
would you mind moving this to a separate PR?

Reply:
Sure.